mithril.js

```diff
@@ -218,9 +218,34 @@
 		return list
 	}
 
+	var applyIEPerf = (
+			typeof document !== "undefined" &&
+			typeof document.documentMode === "number"
+		) ||
+		(
+			typeof navigator !== "undefined" &&
+			typeof navigator.userAgent === "string" &&
+			/\bEdge\/\d/.test(navigator.userAgent)
+		)
+
 	function insertNode(parentElement, node, index) {
-		parentElement.insertBefore(node,
-			parentElement.childNodes[index] || null)
+		if (applyIEPerf) {
+			if (!parentElement._children) {
+				parentElement._children = []
+			}
+			parentElement._children.push(node)
```

Comment:
Does this even work correctly? I'm skeptical whether it does.

Also, batching updates is a thing: it pays off a little on other browsers as well.